scss/tech.scss

```diff
@@ -0,0 +1,67 @@
+.title-tech {
+  font-size: $text-medium;   
+  padding: 10px 10px 10px 10px;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+%size-tech {
+  width: 220px;
+  height: 300px;
+}
+
+%style-border {
+  @extend %size-tech;
+  border: thin solid $earls-green;
+  border-radius: 5px;
+  content: '';
+  position: absolute;
+  z-index: -100;    
+}
+
+.tech-panel {
+  @extend %size-tech;
+  align-items: center;
+  background-color: $earls-green;
+  border: solid $earls-green;
+  border-radius: 5px;
+  flex-direction: column;
+  margin: 20px 20px 20px 20px;
+  padding: 10px 10px 10px 10px;
+  position: relative;
+  text-align: center;
+
+  &::before {
+    @extend %style-border;
+    left: -10px;
+    top: -10px;
+       
+  }  
+
+  &:hover {
+    &::before {
+      @extend %style-border;
```

Comment:
Since the pseudo-element already has it's styles, there's no need to `@extend` all over again :smile_cat: